packages/babel-preset-react-app/index.js

```diff
@@ -115,6 +115,14 @@ module.exports = function(api, opts) {
           regenerator: true,
         },
       ],
+      [
+        require('babel-plugin-named-asset-import'),
+        {
+          loaderMap: {
+            svg: filename => `-!svg-react-loader!${filename}`,
+          },
```

Comment:
Actually, maybe this should be in webpack Babel config and not in our preset? Since it’s directly related to webpack specifically. This also would make a peer dependency on the loader unnecessary (it currently would be).

Reply:
Yeah, either works. The only nice thing about keeping it here is that we don't have to duplicate it in the dev and prod webpack configs.

Reply:
We can solve it in other ways (e.g. by actually starting to modularizing configs a little bit). Doesn't have to be solved here.